src/aliceVision/fuseCut/DelaunayGraphCut.cpp

```diff
@@ -1332,27 +1332,31 @@ bool DelaunayGraphCut::rayCellIntersection(const Point3d& camCenter, const Point
     const Point3d* D = &(_verticesCoords[_tetrahedralization->cell_vertex(tetrahedronIndex, 3)]);
 
     // All the facets of the tetrahedron
-    std::array<std::array<const Point3d*, 3>, 4> facets {{
-        {B, C, D}, // opposite vertex A, index 0
-        {A, C, D}, // opposite vertex B, index 1
-        {A, B, D}, // opposite vertex C, index 2
-        {A, B, C}  // opposite vertex D, index 3
+    std::unordered_map<GEO::index_t, std::array<const Point3d*, 3>> facets{{
```

Comment:
Do not use a more complex/expensive data structure in a performance critical step.
You should remove this commit.

Reply:
done in commit https://github.com/alicevision/AliceVision/pull/832/commits/1d4c33ee323612a7a32aae88fc986cc25365fcfc .